spec/controllers/devices_controllers_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "spec_helper"
+
+describe DevicesController, type: :controller do
+
+  describe 'index' do
+    it '' do
+      get :index
+      expect(response.status).to eq(200)
```

Comment:
existe un shortcut para esto: `expect(response).to have_http_status(200)`
En lugar de usar los números de status hardcodeados rails tiene símbolos para esto `:ok`
http://billpatrianakos.me/blog/2013/10/13/list-of-rails-status-code-symbols/